quiz.js

```diff
@@ -0,0 +1,42 @@
+
+var height =document.getElementById("height");
+var character = document.getElementById("character");
+var treeButton = document.getElementById("keypress-input");
+
+
+function growYourTree(tree) {
+	//console.log("tree", tree );
+		tree = {
+			height: height.value,
+			character: character.value
+		}
+	  
+	  	
+	for (var i = 1; i <= tree.height;  i++ ) {
+		
+	  	var inputChar = tree.character;
+
+		console.log (" ".repeat(height.value--) + inputChar.repeat((i*2)-1));
+ }
+ }
```

Comment:
A lot of random spacing and indentation going on. Please format this is a cleaner way. See me if you need any help